exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Exceptions.kt

```diff
@@ -37,4 +37,14 @@ class ExposedSQLException(cause: Throwable?, val contexts: List<StatementContext
 
 class UnsupportedByDialectException(baseMessage: String, val dialect: DatabaseDialect) : UnsupportedOperationException(baseMessage + ", dialect: ${dialect.name}.")
 
+/**
+ * DuplicateColumnException is thrown :
+ *
+ * When you attempt to create a table with multiple columns having the same name.
+ * When you replace a column of a table so that you define multiple columns having the same name.
+ *
+ * @param columnName the duplicated column name
+ */
+class DuplicateColumnException(columnName: String) : ExceptionInInitializerError("Duplicate column name : $columnName")
```

Comment:
Also, please add a table name to the exception message.

Reply:
Done 👍

Reply:
@Tapac do you think that [THIS](https://github.com/JetBrains/Exposed/pull/719#issuecomment-563425623) feature can be useful?